pkg/controller/cheinstallation/cheinstallation_controller_test.go

```diff
@@ -306,7 +307,8 @@ func TestReconcile(t *testing.T) {
 			AssertThatOperatorGroup(t, cheOperatorNS, OperatorGroupName, cl).Exists()
 			AssertThatSubscription(t, cheOperatorNS, SubscriptionName, cl).Exists()
 			AssertThatCheCluster(t, cheCluster.Namespace, cheCluster.Name, cl).Exists()
-			AssertThatCheInstallation(t, cheInstallation.Namespace, cheInstallation.Name, cl).HasNoCondition()
+			AssertThatCheInstallation(t, cheInstallation.Namespace, cheInstallation.Name, cl).
+				HasConditions(Installing("Status is unknown for CheCluster 'codeready-workspaces'"))
```

Comment:
could you please also add a check (maybe also for the other test-cases when the CheCluster has been created) that the the finalizer (in the `CheInstallation` resource) has been added?

Reply:
good idea! done in 1d2da80

Reply:
I meant to add it to all the test-cases when `CheCluster` exists (in the same way as we always check that the `CheCluster` exists) - one can easily get lost as part of any refactoring effort. 

On the other hand, it's probably fine to have only one check there - so it's up to you ;-)